tests/integration/cli/tests/config.rs

```diff
@@ -42,11 +41,16 @@ fn wasmer_config_error() -> anyhow::Result<()> {
         .lines()
         .map(|s| s.trim().to_string())
         .collect::<Vec<_>>();
+    #[cfg(not(windows))]
+    let expected_1 = "wasmer config --bindir --cflags";
+    #[cfg(windows)]
+    let expected_1 = "wasmer.exe config --bindir --cflags";
```

Comment:
Why we need to do this on this PR?

Reply:
Because I re-enabled integration tests on Windows (necessary to check whether tests pass on Windows) and these tests didn't pass before.